core/arch/arm/include/mm/core_mmu.h

```diff
@@ -239,7 +239,8 @@ struct core_mmu_phys_mem {
 		__register_memory1(#addr, MEM_AREA_SDP_MEM, (addr), (size), \
 				   phys_sdp_mem_section, __COUNTER__)
 #else
-#define register_sdp_mem(addr, size)
+#define register_sdp_mem(addr, size) \
+		static int CONCAT(__unused_var, __COUNTER__) __unused
```

Comment:
I'd still like to have some prefix of the name of the variable that ties it to this macro.

Reply:
Sorry, misunderstanding on my side. Will fix.